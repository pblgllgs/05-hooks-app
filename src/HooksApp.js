import React from "react";

const HooksApp = () => {
    return (
        <>
            <h1>Hola mundo</h1>
        </>
    );
};

export default HooksApp;