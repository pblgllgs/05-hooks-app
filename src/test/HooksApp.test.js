import React from "react";
import { shallow } from "enzyme";
import HooksApp from '../HooksApp'

describe('Pruebas sobre <HooksApp />', () => {

    let wrapper = shallow(<HooksApp />);

    test('Debe de mostrar correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })
})
