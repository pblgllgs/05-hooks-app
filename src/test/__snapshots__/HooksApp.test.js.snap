// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre HooksApp Debe de mostrar correctamente 1`] = `
<Fragment>
  <h1>
    Hola mundo
  </h1>
</Fragment>
`;
